src/app/people/people.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+// Copyright 2019-2020 Google LLC
```

Comment:
If this file is no longer changing, then it shouldn't be part of this commit. Please could you revert it? Thanks!